interfaces/builtin/account_control.go

```diff
@@ -72,6 +72,8 @@ func init() {
 		name:                  "account-control",
 		summary:               accountControlSummary,
 		description:           accountControlDescription,
+		implicitOnCore:        true,
```

Comment:
Is there/will there ever be a case where we have `implicitOnCore: false, implicitOnClassic: true` ? The old code had `implicit` and `implicitOnlyOnClassic` only. Mostly wondering. Also wondering if: `implicit: All`, `implicit: CoreOnly`, `implicit: ClassicOnly` might look nicer (i.e. using bits).